test/integration/rules/aria-prohibited-attr/aria-prohibited-attr.html

```diff
@@ -0,0 +1,40 @@
+<div role="checkbox" aria-required="true" id="pass1">ok</div>
+<div id="pass2" aria-label="  ">Foo</div>
+<div id="pass3" aria-labelledby="  ">Foo</div>
+<div role="alert" aria-selected="true" id="pass4"></div>
+<div role="row" aria-colcount="value" id="pass5"></div>
+
+<div role="caption" aria-label="value" id="fail1"></div>
+<div role="caption" aria-labelledby="value" id="fail2"></div>
+<div role="code" aria-label="value" id="fail3"></div>
+<div role="code" aria-labelledby="value" id="fail4"></div>
+<div role="deletion" aria-label="value" id="fail5"></div>
+<div role="deletion" aria-labelledby="value" id="fail6"></div>
+<div role="emphasis" aria-label="value" id="fail7"></div>
+<div role="emphasis" aria-labelledby="value" id="fail8"></div>
+<div role="insertion" aria-label="value" id="fail9"></div>
+<div role="insertion" aria-labelledby="value" id="fail10"></div>
+<div role="paragraph" aria-label="value" id="fail11"></div>
+<div role="paragraph" aria-labelledby="value" id="fail12"></div>
+<div role="strong" aria-label="value" id="fail13"></div>
+<div role="strong" aria-labelledby="value" id="fail14"></div>
+<div role="subscript" aria-label="value" id="fail15"></div>
+<div role="subscript" aria-labelledby="value" id="fail16"></div>
+<div role="superscript" aria-label="value" id="fail17"></div>
+<div role="superscript" aria-labelledby="value" id="fail18"></div>
+<div aria-label="value" id="fail19"></div>
+<div aria-labelledby="value" id="fail20"></div>
+<!-- technically presentation and none roles do not allow aria-label and aria-labelledby, but since those are global attributes the presentation role conflict will not resolve the roles to none or presentation -->
```

Comment:
Because ARIA loves throwing curve balls, this is no longer true. `aria-label(ledby)` is global "except where prohibited". Which Chromium took to mean that if you use aria-label on role="none", role conflict doesn't trigger. It still does in Firefox. Axe-core would fail it regardless, we're just maybe failing it in the wrong rule?

Maybe we should change this to:

```suggestion
<!- aria-label(ledby) is prohibited on none / presentation. Axe-core considers this to trigger presentation role 
conflict, which was true in ARIA 1.1. This changed in ARIA 1.2 but so far has only been implemented in Chomium. -->
```